californium-core/src/main/java/org/eclipse/californium/core/network/EndpointManager.java

```diff
@@ -48,29 +52,41 @@
  * outbound CoAP requests.
  *
  * The EndpointManager contains the default endpoint for coap (on port 5683) and
- * coaps (CoAP over DTLS). When an application serves only as client but not
- * server it can just use the default endpoint to send requests. When the
+ * endpoint for other schemes (coaps, CoAP over DTLS, coap+tcp, COAP over TCP,
+ * coaps+tcp, COAPS over TCP,). When an application serves only as client but
+ * not server it can just use the default endpoint to send requests. When the
  * application sends a request by calling {@link Request#send()} the send method
  * sends itself over the default endpoint.
  * <p>
  * To make a server listen for requests on the default endpoint, call
- * <pre>{@code
- *  CoapServer server = new CoapServer();
- * }</pre>
+ * 
+ * <pre>
+ * {
+ * 	&#064;code
```

Comment:
I don't need the `<pre>`. but its more a relict and just listed as difference caused be source formatting. 
Remove or replace by the old one?